src/main.c

```diff
@@ -127,16 +127,15 @@ main (int argc, char *argv[])
 
   }
   while (next_option != -1);
-
-  if (getenv ("HOME") == NULL)
+  
+  char *HOMEDIR = getenv ("HOME");
+  
+  if (HOMEDIR == NULL)
```

Comment:
Right, but the else is part of the first if, so the block below it should be indented 2 spaces to the left.